pkg/monitor/cluster/clusterflagsandbanner.go

```diff
@@ -0,0 +1,46 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
+)
+
+const (
+	operatorFlagMetricsTopic  = "cluster.nonstandard.operator.featureflags"
+	supportBannerMetricsTopic = "cluster.nonstandard.contact.support.banner"
+)
+
+func (mon *Monitor) emitOperatorFlagsAndSupportBanner(ctx context.Context) error {
+	cs, err := mon.listAROClusters(ctx)
+	if err != nil {
+		return err
+	}
+
+	for _, cluster := range cs.Items {
+		if cluster.Spec.OperatorFlags != nil {
```

Comment:
what if operatorFlags are nil? should we do anything about that? do we care?

Reply:
actually I am not 100% sure about this since I believe some of the old-version clusters do not have any valid operator flags, should we regard them as unsupported ones or not?